.goreleaser.yml

```diff
@@ -24,4 +24,7 @@ snapshot:
   name_template: "{{ .Tag }}-SNAPSHOT"
 
 checksum:
-  name_template: 'checksums.txt'
\ No newline at end of file
+  name_template: 'checksums.txt'
+
+changelog:
```

Comment:
How does goreleaser know the tag?  Does it use the latest tag always?  If so we'd have to make sure to tag before merging CHANGELOG.md changes to main, or do some automated tagging as suggested above.

Reply:
We set the envvar `GORELEASER_CURRENT_TAG` to what changie tells us is the latest version (i.e. the one we just changed CHANGELOG.md for)